powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OfflineSignatureServiceBehavior.java

```diff
@@ -336,10 +338,10 @@ private VerifyOfflineSignatureResponse verifyOfflineSignatureImpl(final VerifyOf
         }
     }
 
-    private boolean processProximityCheck(final VerifyOfflineSignatureParameter request, final int digitsNumber) throws CryptoProviderException {
+    private VerifyOfflineSignatureResponse.ProximityCheck processProximityCheck(final VerifyOfflineSignatureParameter request, final int digitsNumber) throws CryptoProviderException {
         if (StringUtils.isBlank(request.getProximityCheckSeed())) {
             logger.debug("Proximity seed is not present and is TOTP not being verified, activation ID: {}", request.getActivationId());
-            return true;
+            return null;
```

Comment:
I would add a comment here, just to emphasize the difference between returning `proximityCheck.success=true`.